Source/BlueprintSearchVSExtension/Source/BlueprintSearchVSPackage.cs

```diff
@@ -35,8 +42,73 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 		{
 			// When initialized asynchronously, the current thread may be a background thread at this point.
 			// Do any initialization that requires the UI thread after switching to the UI thread.
+			await base.InitializeAsync(cancellationToken, progress);
 			await this.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 			await BlueprintSearchVSWindowCommand.InitializeAsync(this);
+			await InitDTEAsync();
+			InitSolutionLoaderManager();
+		}
+
+		private async Task InitDTEAsync()
+		{
+			Commands.CommandHelpers.PathFinderHelper.DTEService = await GetServiceAsync(typeof(DTE)) as DTE2;
+		}
+
+		private void InitSolutionLoaderManager()
+		{
+			SolutionLoaderManager = GetService(typeof(SVsSolution)) as IVsSolution;
+			SolutionLoaderManager.AdviseSolutionEvents(this, out SolutionEvents);
```

Comment:
Do you need to keep a copy to this SolutionLoaderManager? It's not referenced in any other function, so you could simplify this for now to:
```suggestion
		((IVsSolution)GetService(typeof(SVsSolution))).AdviseSolutionEvents(this, out SolutionEvents);
```